nat-lab/docker-compose.yml

```diff
@@ -36,6 +36,8 @@ services:
     volumes:
       - ./:/sync
       - ../:/libtelio
+    healthcheck:
+      test: "ls /ready"
```

Comment:
Shouldn't this healthcheck be added to all of the containers?

Reply:
good point!

Reply:
Fixed